docs/source/testing/integration.rst

```diff
@@ -18,4 +18,135 @@
 Writing Integration Tests for NWChemEx
 ######################################
 
-TODO: Write me!!!!
+NWChemEx is a modular ecosystem designed with separation of concerns as a key
+design point. An example of this separation can be found with the 
+SCF, integrals, ChemCache libraries. These components of NWX are linked by 
```

Comment:
To keep consistency, better to use "NWChemEx" instead of "NWX" here.

Reply:
I think "NWX" should be fine. We use it has an abbreviation for "NWChemEx" regularly, including other places in these docs.